app/src/marketplace/containers/ProductPage/MembersGraphV2.jsx

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import React, { useEffect, useState, useMemo } from 'react'
+
+import ClientProvider from '$shared/components/StreamrClientProvider'
+import TimeSeriesGraph from '$shared/components/TimeSeriesGraph'
+import { getJoinsAndParts } from '$mp/modules/dataUnion/services'
+
+type Props = {
+    dataUnionAddress: string,
+    memberCount: number,
+    shownDays?: number,
+}
+
+const MILLISECONDS_IN_DAY = 24 * 60 * 60 * 1000
+
+const MembersGraphV2 = ({ dataUnionAddress, memberCount, shownDays = 7 }: Props) => {
```

Comment:
Could rename the old one as `deprecated_MembersGraph` like in the services... but good as it is as well 👍